src/Umbraco.Web/PublishedContentExtensions.cs

```diff
@@ -174,7 +174,7 @@ public static T Value<T>(this IPublishedContent content, string alias, string cu
 
             // else... if we have a property, at least let the converter return its own
             // vision of 'no value' (could be an empty enumerable) - otherwise, default
-            return property == null ? default : property.Value<T>(culture, segment);
+            return property == null ? default : property.Value<T>(culture, segment, defaultValue: defaultValue);
```

Comment:
Shouldn't we return defaultValue in case the property is null as well? Or do we want to make a difference between "property exists but has no value" and "property does not exist"?

Reply:
Yes good catch